txpool/txpool.go

```diff
@@ -177,6 +186,7 @@ func NewTxPool(
 	network *network.Server,
 	metrics *Metrics,
 	config *Config,
+	whitelists Whitelists,
```

Comment:
Why not put this in the TxPool config, like `PriceLimit` and `Sealing`?

Reply:
cc @dbrajovic

Reply:
fixed 6974e01cbb5bd4d4c82039d9a5decc59aba85c87